docs/user/getting-started.md

```diff
@@ -199,52 +199,51 @@ Finally, move it into a location within your shell's search path.
 Retrieve the password for the "demouser" account that's been automatically generated for you during installation:
 
 ```console
-$ kubectl get secret/keycloak-demouser-secret -n flightctl -o=jsonpath='{.data.password}' | base64 -d
-
-[...]
+password=$(kubectl get secret/keycloak-demouser-secret -n flightctl -o=jsonpath='{.data.password}' | base64 -d)
 ```
 
-Use the CLI to log into the Flight Control Service:
+Headless login:
 
 ```console
-$ flightctl login https://api.flightctl.127.0.0.1.nip.io/ --web --insecure-skip-tls-verify
-
-[...]
+flightctl login <flightctl_api_url> -u demouser -p ${password}
 ```
 
-In the web browser that opens, use the login "demouser" and the password you retrieved in the previous step.
+> 📌 For headless login to work, the OIDC provider of your choice needs to have Direct Access grant enabled
 
-Verify you can now access the service via the CLI:
+Login using web browser:
 
 ```console
-$ flightctl get devices
-
-NAME                                                  OWNER   SYSTEM  UPDATED     APPLICATIONS  LAST SEEN
+flightctl login <flightctl_api_url> --web
 ```
 
 ### ACM deployment
 
-Use the CLI to log into the Flight Control Service:
+You will be logging in with ACM login credentials.
 
-```console
-$ flightctl login https://api.flightctl.127.0.0.1.nip.io/ --web --insecure-skip-tls-verify
+Headless login:
 
-[...]
+```console
+flightctl login <flightctl_api_url> -u <acm_user> -p <password>
 ```
 
-In the web browser that opens, use your ACM login credentials.
-
-Verify you can now access the service via the CLI:
+Login using a user token:
 
 ```console
-$ flightctl get devices
+flightctl login <flightctl_api_url> -t $(oc whoami -t)
```

Comment:
Instead of listing all options, should we only list this one, which for an ACM user seems to be the most reliable?
Is there any benefit to the other ones? Ok, maybe that you do not need to login via `oc` first.

Reply:
I agree with the `-t` being the best. Since the guide goes through the helm install process too, i'd say we are already logged it via `oc`.